apps/remote_control/test/lexical/remote_control/build/error_test.exs

```diff
@@ -92,11 +143,13 @@ defmodule Lexical.RemoteControl.Build.ErrorTest do
         |> compile()
         |> parse_error()
 
-      assert [error, detail] = errors
+      # assert [error, detail] = errors
+      assert [detail, error] = errors
+
       assert error.message =~ "unexpected reserved word: end"
       assert error.position == {15, 9}
 
-      assert detail.message =~ ~S[The "(" here is missing terminator ")"]
+      assert detail.message =~ ~S["(" here is missing terminator ")"]
```

Comment:
why change the assertion, just downcase the message with the old assertion.

Reply:
because they changed `The` to `the`

Reply:
ok